Xamarin.Forms.Core/Shapes/Line.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Xamarin.Forms.Platform;
+
+namespace Xamarin.Forms.Shapes
+{
+    [RenderWith(typeof(_LineRenderer))]
+    public sealed class Line : Shape
+    {
+        public static readonly BindableProperty X1Property =
+            BindableProperty.Create(nameof(X1), typeof(double), typeof(Line), 0.0d);
+
+        public static readonly BindableProperty Y1Property =
+            BindableProperty.Create(nameof(Y1), typeof(double), typeof(Line), 0.0d);
+
+        public static readonly BindableProperty X2Property =
+            BindableProperty.Create(nameof(X2), typeof(double), typeof(Line), 0.0d);
+
+        public static readonly BindableProperty Y2Property =
```

Comment:
it's quite inconsistent to have Line defined with 2 points, and other shapes with width and height